MAINTENANCE.md

```diff
@@ -113,7 +114,12 @@ a **New upload** for it inside the
     ```
 
 6. Add a link to the new release version documentation in `README.rst`.
-7. Open a new PR with the updated changelog.
+7. Update DOI badge in `README.rst`.
+8. Update citation on `CITATION.rst`.
+8. Find any appearance of the old DOI and check if it needs to be updated or
+   the new one must be added. You can perform the search through
+   `git grep <OLD-DOI>`.
```

Comment:
This is only necessary if we're including the version specific DOI in these places. For Verde and Pooch, we have the JOSS papers so this isn't necessary. Might be best to include this in a separate checklist for this particular case.

Reply:
You're right!